docs/node/admin/kwil-admin/snapshot/index.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 0
+sidebar_label: "snapshot"
+id: "kwil-admin-snapshot"
+title: "kwil-admin snapshot"
+description: "The `snapshot` command is used to create and manage snapshots."
+slug: /ref/kwil-admin/snapshot
+---
+
+## kwil-admin snapshot
+
+The `snapshot` command is used to create and manage snapshots.
+
+### Synopsis
+
+The `snapshot` command is used to create and manage snapshots.
+
+### Options
+
+```
+  -h, --help   help for snapshot
+```
+
+### Options inherited from parent commands
+
+```
+      --output string   the format for command output - either 'text' or 'json' (default "text")
+  -S, --silence         Silence logs
+```
+
+### SEE ALSO
```

Comment:
For the category "index.md" page, we can use react cards like on https://docs.kwil.com/docs/admin.

![image](https://github.com/kwilteam/docs/assets/140431406/fb6a8d44-7b08-4104-90d0-86245560c281)


I think that should work if the hierarchy is established correctly.